index.html

```diff
@@ -10,22 +10,150 @@
 			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
 			rel="stylesheet"
 		/>
+		<link rel="stylesheet" href="./css/normalize.css" />
 		<link rel="stylesheet" href="css/global.css" />
+		<link rel="stylesheet" href="./css/header.css" />
 		<!-- In a new branch, rename the page.css file adding the name of that actual page (Example: home.css) -->
 		<link rel="stylesheet" href="css/page.css" />
 	</head>
 
 	<body>
 		<!-- Header Navigation-->
+		<header>
+			<!-- nav -->
+			<nav class="top-bar">
+				<div class="dp-wrapper">
+					<!-- logo container -->
+					<div class="logo-container">
+						<div class="img-container logo-icon">
+							<picture>
+								<source
+									media="(min-width:768px)"
+									srcset="./assets/icons/global/logo-icon-big.svg"
+								/>
+								<img
+									src="./assets/icons/global/logo-icon.svg"
+									alt="If These Lands Could Talk logo-icon"
+									style="width: auto"
+								/>
+							</picture>
+						</div>
+						<div>
+							<!-- waiting for sr class -->
+							<!-- <h1 class="sr-only">If These Lands <span class="block"> Could Talk</span></h1> -->
+						</div>
+					</div>
+					<!-- end logo -->
+					<!-- desktop menu -->
+					<div class="desktop-menu">
+						<ul class="desktop-nav">
+							<li>
+								<a href="#"> About Us</a>
```

Comment:
Please add the paths to the page files to the hrefs. I already had the footer team do this, so you can use their work for reference: https://github.com/indigenousfriends/itlct-website-redesign/blob/footer/index.html

Reply:
changed the placeholder links to html files !